editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -951,8 +953,12 @@ const PullRequestBlock = () => {
 export const GithubPane = React.memo(() => {
   const githubUser = useEditorState(
     (store) => store.editor.githubData.githubUserDetails,
-    'Github user details',
+    'GithubPane githubUser',
   )
+  const isLoggedIn = useEditorState((store) => {
```

Comment:
@maltenuhn I went in and split these into two useEditorState hooks. the reason is that the original githubUser hook may benefit from shallow equality checks, so putting it one level deeper may blow the equality and thus trigger rerenders of this component.